add section on moving into our organization (#1345)
nextcloud/documentation: developer_manual/app/publishing.rst

* App authors must provide means to contact them, be it through a bug tracker, forum or mail.

Apps which break the guidelines will lose their 'approved' or 'official' state; and might be blocked from the app store altogether. This also has repercussions for the author, especially in case of security concerns, he/she might find themselves blocked from submitting applications.

> Putting your repo in the Nextcloud Project

Reviewer: ```suggestion
Moving your repo to the Nextcloud organization
```

------------------------------------------

> We're always delighted to hear app develpers are interested in moving their app into the Nextcloud Project in `github.com/nextcloud <https://github.com/nextcloud>`_! There are benefits for users and developers in being there. However, it comes with some requirements as well.

Reviewer: ```suggestion
We're always delighted to hear app developers are interested in moving their app to the Nextcloud organization at `github.com/nextcloud <https://github.com/nextcloud>`_! There are benefits for users and developers in being there. However, it comes with some requirements as well.
```

Benefits
^^^^^^^^

> * You can use the tools and bots we have set up, including translations and such

Reviewer 2: translation sync is also possible when being outside of our repo.

> * Everybody in the Nextcloud project can more easily contribute

Reviewer: ```suggestion
* Everybody in the Nextcloud organization can contribute more easily
```

* Your visibility to app developers increases
* Users can expect apps in our project to be better maintained

Requirements
^^^^^^^^^^^^
To deliver on the promises above, we have two simple rules.

> * You participate in our `collaborative development model <../general/code-of-conduct.html>`_

Reviewer: ```suggestion
* You work and communicate according to the values of our `Code of Conduct <../general/code-of-conduct.html>`_
```

* When you are no longer active, our admins can decide to hand over maintainership to another contributor

We want to make sure that when you find other things in life which are more urgent or otherwise are unable to help your project anymore, it does not become 'dead code' as long as there are people who want to keep it alive. This is not fair to users, who would be forced to remove the app and install another.

> Please note that the role of a maintainer is not to be the most active or prolific contributor to a project! Worlds' `most famous maintainer of an open source project <https://en.wikipedia.org/wiki/Linus_Torvalds>`_ has not appeared in the top-100 of contributors by lines of code for decades, yet he runs the largest single software development project in the history of mankind. Being friendly, welcoming and responsive are what it takes to be a successful maintainer. Not being the most brilliant developer ever, or spending nights and weekends coding!

Reviewer: I’d prefer we don’t put Linus Torvalds even more on a pedestal. :) We don’t really need a specific example there.

Reviewer: ```suggestion
Please note that the role of a maintainer is not to be the most active or prolific contributor to a project! Being friendly, welcoming and responsive are what it takes to be a successful maintainer. Not being the most brilliant developer ever, or spending nights and weekends coding!
```

> The goal of these rules is simple: help your project be more successful. We also suggest you watch this talk by `Jan about building a great community. <https://www.youtube.com/watch?v=UtAoRIKVpW4>`_

Reviewer: :see_no_evil:

How to move
^^^^^^^^^^^

> To move into our github repo, just ask any of our contributors, `especially those who are admin. <https://github.com/orgs/nextcloud/people?utf8=%E2%9C%93&query=+role%3Aowner>`_ They will be happy to help!

Reviewer: ```suggestion
To move to our Github organization, just ask any of our contributors, `especially those who are admin. <https://github.com/orgs/nextcloud/people?utf8=%E2%9C%93&query=+role%3Aowner>`_ They will be happy to help!
```
